[PLINT-524] Sending data from vSAN to EvP intake (#19070)
DataDog/integrations-core: vsphere/README.md

Property metrics are prefixed by the resource name. For example, host property metrics metrics are prefixed with `vsphere.host.*`, and VM property metrics are prefixed with `vsphere.vm.*`. View all the possible property metrics in the [metadata.csv][10].

#### Resource Catalog metrics

If the `collect_vsan_data:true` config option is set, then discovered vSAN clusters 

> will be visualized as resources in the Resource Catalog product, with the following

Reviewer: ```suggestion
are visualized as resources in the Resource Catalog product, with the following
```
